source/_integrations/voicerss.markdown

```diff
@@ -3,6 +3,7 @@ title: VoiceRSS
 description: Instructions on how to setup VoiceRSS TTS with Home Assistant.
 ha_category:
   - Text-to-speech
+ha_iot_class: Cloud Polling
```

Comment:
This integration doesn't poll on a regular interval. It immediately gets the requested TTS on demand. Cloud Push is more fitting in this case, as we push out the requested information and receive it directly.

```suggestion
ha_iot_class: Cloud Push
```

Reply:
> This integration doesn't poll on a regular interval. It immediately gets the requested TTS on demand. Cloud Push is more fitting in this case, as we push out the requested information and receive it directly.

Thank you for your feedback and clarification, @frenck! I understand your point and I agree with it.